motionSetPointsApp/src/motionSetPoints.h

```diff
@@ -38,6 +38,10 @@ class motionSetPoints : public asynPortDriver
 #define FIRST_MSP_PARAM P_positions
 #define LAST_MSP_PARAM P_coordRBV
 
+    // File information.
+    int P_fileName; // string - motio setpoints file name
+    int P_errorMsg; // string - error message
+
```

Comment:
This has been added after `P_coordRBV` and so should become `LAST_MSP_PARAM` however this concept is no longer needed so for clarity we should remove  FIRST_MSP_PARAM , LAST_MSP_PARAM  and the extra asyn constructor argument they are are passed to.